src/main/resources/reference.conf

```
@@ -1,590 +1,595 @@
+      strings: {
+        format: "csv"
+        path: ${common.path}"/input/interaction/string-interactions.txt.gz"
+        options: [
+          { k: "sep",    v: " "  }
+          { k: "header", v: true }
         ]
       }
     }
-    output = {
-      interactions {
-        format = ${common.output_format}
-        path = ${common.path}"/output/interaction"
+    output: {
+      interactions: {
+        format: ${common.output_format}
+        path: ${common.path}"/output/interaction"
       }
-      interactions_evidence {
-        format = ${common.output_format}
-        path = ${common.path}"/output/interaction_evidence"
+      interactions_evidence: {
+        format: ${common.output_format}
+        path: ${common.path}"/output/interaction_evidence"
       }
-      interactions_unmatched {
-        format = ${common.output_format}
-        path = ${common.path}"/excluded/interaction"
+      interactions_unmatched: {
+        format: ${common.output_format}
+        path: ${common.path}"/excluded/interaction"
       }
     }
   }
 
-  target {
-    input {
-      chembl = {
-        format = "json"
-        path = ${common.path}"/input/target/chembl/chembl_target.jsonl"
-      }
-      chemical_probes = {
-        format = "json"
-        path = ${common.path}"/input/target/chemicalprobes/chemicalprobes.json.gz"
-      }
-      diseases = {
-        format = "parquet"
-        path = ${common.path}"/output/disease"
-      }
-      ensembl {
-        format = ${common.output_format}
-        path = ${common.path}"/intermediate/target/ensembl/homo_sapiens.parquet"
-      }
-      gene_code = {
-        format = "csv"
-        path = ${common.path}"/input/target/gencode/gencode.gff3.gz"
-        options = [
-          {k: "comment", v: "#"}
-          {k: "sep", v: "\\t"}
+  target: {
+    input: {
+      chembl: {
+        format: "json"
+        path: ${common.path}"/input/target/chembl/chembl_target.jsonl"
+      }
+      chemical_probes: {
+        format: "json"
+        path: ${common.path}"/input/target/chemicalprobes/chemicalprobes.json.gz"
+      }
+      diseases: {
+        format: "parquet"
+        path: ${common.path}"/output/disease"
+      }
+      ensembl: {
+        format: ${common.output_format}
+        path: ${common.path}"/intermediate/target/ensembl/homo_sapiens.parquet"
+      }
+      gene_code: {
+        format: "csv"
+        path: ${common.path}"/input/target/gencode/gencode.gff3.gz"
+        options: [
+          { k: "sep",     v: "\\t" }
+          { k: "comment", v: "#"   }
         ]
       }
-      genetic_constraints = {
-        format = "csv"
-        path = ${common.path}"/intermediate/target/gnomad/gnomad_lof_by_gene.txt.gz"
-        options = [
-          {k: "sep", v: "\\t"}
-          {k: "header", v: true}
-          {k: "nullValue", v: "NA"}
+      genetic_constraints: {
+        format: "csv"
+        path: ${common.path}"/intermediate/target/gnomad/gnomad_lof_by_gene.txt.gz"
```

```suggestion
        path: ${common.path}"/input/target/gnomad/gnomad.v4.1.constraint_metrics.tsv"
```

Don't forget to update the actual input!

---

applied